### ISSUE TITLE
Funding

### ISSUE THREAD

**Issue by dankohn** [cncf/landscape#1023]
The funding page has shown 3 updates for 12/1 on 12/1, then the same 3 on 12/2, then the same 3 on 12/3. There's a logic error where it's not correctly putting the updates on the date where they first occurred.

**Comment by ZeusTheTrueGod** [comment]
Hard to guess, but I've added a check that we only use commits from the 'CNCF-bot', otherwise a processed_landscape.yml file could be out of sync

**Comment by ZeusTheTrueGod** [comment]
@dankohn bug has been fixed, and I've notified netlify about issues on their side.
1) Somehow their repo did not have any remotes, i.e. there was not `origin/master` at all
2) Also the working directory reported that we are on a latest commit from Nov, 19 , but the working directory had a proper content for the recent commit. That issue happened only on the master branch, other branches had proper logs for the master branch.

The solution was to add `origin` remote with a REPOSITORY_URL and do not fail if that remote already exists. Then I started to use logs from origin/master instead of master, because origin/master has proper logs.

It seemed like running `git remote add origin ...` fixed the problem with a missing remote, still I keep the code which will add that missing origin remote again if it is lost.
